lib/features/replace/BpmnReplace.js

```diff
@@ -199,6 +199,18 @@ export default function BpmnReplace(
       // else if property is explicitly set, use it
       else if (target && has(target, 'isExpanded')) {
         newElement.isExpanded = target.isExpanded;
+
+        // assign default size of new expanded element
+        const defaultSize = elementFactory.getDefaultSize(newBusinessObject, {
```

Comment:
No const with PhantomJS ;)
```suggestion
        var defaultSize = elementFactory.getDefaultSize(newBusinessObject, {
```

Reply:
Man, I've been working with our ES6 projects for too long. 🤡

Reply:
Fixed.